sharktank/sharktank/utils/load_llm.py

```diff
@@ -100,6 +101,9 @@ def begin_batch(
         cache_state = self.model.cache.allocate(self.page_cache_size)
         self.free_pages = list(range(1, self.page_cache_size))
 
+        if max_decode_steps < dump_decode_steps:
+            max_decode_steps = dump_decode_steps
+
```

Comment:
If anything this should throw as this is an impossible request.

Reply:
Max decode steps is useful. It was useful for me when I wanted to run just one step.

Reply:
I guess we could increase the default CLI value.